Userland/Services/Taskbar/main.cpp

```diff
@@ -54,7 +75,11 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     auto menu = TRY(build_system_menu());
     menu->realize_menu_if_needed();
 
-    auto window = TRY(TaskbarWindow::try_create(move(menu)));
+    auto window = TRY(Taskbar::TaskbarWindow::try_create(move(menu)));
+
+    Config::monitor_domain("Taskbar");
```

Comment:
Would it make sense to put this in the `TaskbarChangeListener` constructor instead, since it logically pairs with that class?